### PR TITLE
DMCA notes and template

### PR DESCRIPTION
**Pull request by RiversideRocks**

### CODE REVIEW

File: DMCA.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Sample DMCA
+
+```
+Hello,
+
+I am writing on behalf of {{website}}. I am the webmaster of {{website}} and all of its subdomains.
+
+I see you are filing a claim for {{invidious instance url}}. {{invidious instance url}} hosts Invidious, a private YouTube front-end, meaning it is simply a proxy to access YouTube assets and user generated content without tracking from Google. Thus, all content is mirrored from YouTube and is not stored on our servers and if YouTube chooses to remove an asset, it will no longer appear on our site. Under United States copyright law 17 U.S.C. § 512(a), part of the Digital Millennium Copyright Act, we cannot be liable for content that "is transmitted through the system or network without modification of its content." It should be noted our OVH server hosting {{invidious instance url}} is hosted in {{Country}}.
```
Comment (unixfox) [inline DMCA.md:20]: `OVH server hosting` to just `server hosting`?

**Comment by unixfox** [comment]
I just wanted to say thank you for sharing this material, it's very useful.

**Comment by RiversideRocks** [comment]
Fixed, thank you!

**Comment by SamantazFox** [comment]
I think that "my instance is hosted in {{ country }}" (i.e DMCA doesn't apply) and US specifics should be two different things. Also, could you please make the filename more generic (like `takedown_requests`) with a dedicate DMCA section? That way, it could be further expanded for other types of takedowns (E.g under EU laws).

**Comment by RiversideRocks** [comment]
I've made some updates. Would an invidious member from the EU mind creating a sample template for Europe based servers? I'm afraid I'm not familiar with EU laws.

**Comment by SamantazFox** [comment]
> I've made some updates. Would an invidious member from the EU mind creating a sample template for Europe based servers? I'm afraid I'm not familiar with EU laws.

Will probably do that soon, if I have the time, yeah :)

**Comment by RiversideRocks** [comment]
Could we merge this for now and add the EU notice later?

**Comment by unixfox** [comment]
> Could we merge this for now and add the EU notice later?

Sure but then we should make a github issue in order to not forget about it ;)

**Pull request merged**